.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,6 +18,16 @@ can uncomment the below line to indicate which issue your PR fixes, for example
 Fixes #
 -->
 
+### Testing done
+<!--
+How has this code been tested?
```

Comment:
Perhaps this should be the title of this section, rather than "How has this code been tested?"

Reply:
I've been using a Markdown Linter that tends to disagree.
https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md026---trailing-punctuation-in-header
I've been blindly abiding, but have created an issue there to suggest they offer some rationale for this ruling.